packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.styles.tsx

```diff
@@ -11,7 +11,7 @@ import { ISearchBoxProps, ISearchBoxStyleProps, ISearchBoxStyles } from './Searc
 
 export function getStyles(props: ISearchBoxStyleProps): ISearchBoxStyles {
   const { theme, underlined, disabled, hasFocus, className, hasInput } = props;
-  const { palette, fonts } = theme;
+  const { palette, fonts, semanticColors } = theme;
```

Comment:
shouldn't you get a compiler warning for adding semanticColors here but not use it?

Reply:
true! I just added the slot for inputPlaceholderText.